script/foreman-debug

```diff
@@ -39,20 +39,20 @@ Please note that the rsync transmission is UNENCRYPTED.
 UPLOADUSAGE
 }
 
-# filter for patterns like password=XYZ or secret: abc!@#$123
+# Filter for patterns like password=XYZ, -storepass XYZ or secret: XYZ
 FILTER_WORDS=(
 password
 PASSWORD
 default_password
+key
```

Comment:
Yes, the problem with current approach mentioning only specific strings was not working apparently, this patch tighten the filtering causing a lot more false alerts which should be safer. If you don't like, I can simply name them.